pages/maker/apps/index.tsx

```diff
@@ -1,45 +1,40 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import Head from '@containers/head'
 import { bindActionCreators } from 'redux'
-import { connect } from 'react-redux'
+import { connect, useSelector } from 'react-redux'
 
 import { AppDirectory } from '@components/app-directory'
 import { Flex, Box } from '@blockstack/ui'
 import { isUserSignedIn } from '@stores/user/selectors'
 import UserStore from '@stores/user'
+import { selectAppList } from '@stores/maker/selectors'
 
 import { selectApiServer } from '@stores/apps/selectors'
 import { Page } from '@components/page'
 import { SignIn } from '@components/sign-in'
 
-const SAMPLE_APPS = [
-  {
-    name: 'Sigle',
-    id: 28
-  },
-  {
-    name: 'XorDrive',
-    id: 2
-  }
-]
-
 interface AppDirectoryPageProps {
-  isSignedIn: boolean;
-  apiServer: string;
   handleSignIn(server: string): any;
   signIn: any;
 }
 
-const AppDirectoryPage: React.FC<AppDirectoryPageProps> = ({ isSignedIn, apiServer, signIn, handleSignIn }) => {
-  handleSignIn(apiServer)
+const AppDirectoryPage: React.FC<AppDirectoryPageProps> = ({ signIn, handleSignIn }) => {
+  const { apps, isSignedIn, apiServer } = useSelector(state => ({
```

Comment:
This is the [redux hooks API](https://react-redux.js.org/next/api/hooks) - I find it much cleaner than the `connect` pattern, and requires much less boilerplate, while still being type-safe. We are using this pattern with redux on newer projects, so if you'd like to use this moving forward, I think we'd encourage it.